Prometheus.AspNetCore.Grpc/Prometheus.AspNetCore.Grpc.csproj

```diff
@@ -16,22 +17,13 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
     <NoWarn>1701;1702;1705;1591</NoWarn>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-    <DocumentationFile>bin\Release\netstandard2.0\Prometheus.Grpc.xml</DocumentationFile>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <DocumentationFile>bin\Debug\netstandard2.0\Prometheus.Grpc.xml</DocumentationFile>
+    <IsPackable>true</IsPackable>
+    <OutputType>Library</OutputType>
```

Comment:
Adding these two lines allows `dotnet pack` to work in conjunction with the web SDK.